_articles/security/where-is-data-stored-computer.md

```diff
@@ -36,6 +36,7 @@ You can override the storage location for your Bitwarden desktop application dat
   - Vivaldi: `%LocalAppData%\Vivaldi\User Data\Default\Local Extension Settings\nngceckbapebfimnlniiiahkandclblb`
   - Brave: `%AppData%\brave\Local Extension Settings\nngceckbapebfimnlniiiahkandclblb`
   - Edge: `%LocalAppData%\Packages\Microsoft.MicrosoftEdge_8wekyb3d8bbwe\AC\MicrosoftEdge\Extensions\Storage`
+  - Chromium based Edge: `%AppData%\Local\Microsoft\Edge\User Data\Default\Extensions\jbkfoedolllekgbhcbcoahefnbanhhlh`
```

Comment:
Let's just replace the "old edge" since it is going away.


```suggestion
  - Edge: `%AppData%\Local\Microsoft\Edge\User Data\Default\Extensions\jbkfoedolllekgbhcbcoahefnbanhhlh`
```